curp/src/client.rs

```diff
@@ -542,6 +545,10 @@ where
                         continue;
                     }
                 };
+                if inner.members.is_empty() {
+                    // ignore the empty members response
+                    continue;
+                }
```

Comment:
The `ok_cnt` is not checked at the end of this method, and it skipped adding because of this continue.

Reply:
fixed